mbed_devices/_internal/windows/windows_identifier.py

```diff
@@ -0,0 +1,68 @@
+"""Defines a Windows identifier."""
+from typing import NamedTuple, Any, Optional
+
+from mbed_devices._internal.windows.component_descriptor_utils import is_undefined_value
+
+INSTANCE_ID_CHARACTER = "&"
+
+
+class WindowsUID(NamedTuple):
+    """Definition of a Windows Universal ID."""
```

Comment:
I'm not sure what a "Windows Universal ID" is. Might be worth adding a description or link to existing docs here.

Reply:
tried to comment more about this 28301b80044ce0a6df8120eed868e2979d6718dd